index.js

```diff
@@ -236,17 +236,22 @@ function getter(options) {
 				// since Google doesn't put there '.svg' suffix for some reason
 				// (but does that for other extensions).
 				var re = new RegExp([
-					"\\s*font-family:\\s*'([^']+)';",
-					"\\s*font-style:\\s*(\\w+);",
-					"\\s*font-weight:\\s*(\\w+);",
-					"\\s*src:([^;]*);",
-					".*(?:unicode-range:([^;]+);)?",
+					"\\s*font-family:\\s*'(?<family>[^']+)';",
+					"\\s*font-style:\\s*(?<style>\\w+);",
+					"\\s*font-weight:\\s*(?<weight>\\w+);",
+					".*(?:\\s*font-stretch:\\s(?<stretch>[^;]+);)?",
+					"\\s*src:(?<src>[^;]*);",
+					".*(?:unicode-range:(?<range>[^;]+);)?",
 				].join(''), 'm');
 
-				return formatData.apply(null, block.match(re, 'm'));
-
+				let found = block.match(re, 'm');
+				if (found === null) {
+					throw new Error("faild to match block");
```

Comment:
Typo, "faild" => "failed"

Reply:
Thanks. It's fixed in https://github.com/battlesnake/gulp-google-webfonts/pull/57 .